common/src/main/java/com/wynntils/utils/type/AnimationPercentage.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.utils.type;
```

Comment:
I am pretty certain bolyai said `utils.render.type`. That is where it belongs, anyway.